.github/workflows/native-test-e2e-android.yml

```diff
@@ -0,0 +1,138 @@
+name: "[Test] suite-native Android E2E"
+on: [pull_request, workflow_dispatch]
```

Comment:
a little nitpick, but I like this formatting better
```yaml
on:
  pull_request:
  workflow_dispatch:
```

Reply:
fixed by 5d9543b375